data/KishanGupta.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Kishan Gupta",
+  "type": "personal",
+  "bio": "A Full Stack Developer and Looking for contribute in Open Source Projects.",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/i_kishan_",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/i-kishan",
+      "icon": "FaLinkedIn"
```

change the icon to `SiLinkedin`
```suggestion
      "icon": "SiLinkedin"
```